core/test/usAnyTest.cpp

```diff
@@ -114,24 +114,28 @@ int usAnyTest(int /*argc*/, char* /*argv*/[])
   US_TEST_CONDITION(anyListOfAnys.ToString() == "[1,hello]", "Any[std::list<Any>].ToString()")
   US_TEST_CONDITION(anyListOfAnys.ToJSON() == "[1,\"hello\"]", "Any[std::list<Any>].ToJSON()")
 
-  std::map<std::string, int> map1;
+  std::unordered_map<std::string, int> map1;
   map1["one"] = 1;
   map1["two"] = 2;
   Any anyMap1 = map1;
-  US_TEST_CONDITION(anyMap1.Type() == typeid(std::map<std::string, int>), "Any[std::map<std::string,int>].Type()")
-  US_TEST_CONDITION((any_cast<std::map<std::string, int> >(anyMap1) == map1), "any_cast<std::map<std::string,int>>()")
-  US_TEST_CONDITION(anyMap1.ToString() == "{one : 1, two : 2}", "Any[std::map<std::string,int>].ToString()")
-  US_TEST_CONDITION(anyMap1.ToJSON() == "{\"one\" : 1, \"two\" : 2}", "Any[std::map<std::string,int>].ToJSON()")
-
-  std::map<int, Any> map2;
+  US_TEST_CONDITION(anyMap1.Type() == typeid(std::unordered_map<std::string, int>), "Any[std::unordered_map<std::string,int>].Type()")
+  US_TEST_CONDITION((any_cast<std::unordered_map<std::string, int> >(anyMap1) == map1), "any_cast<std::unordered_map<std::string,int>>()")
+  std::string anyMap1_ToString(anyMap1.ToString());
+  std::string anyMap1_ToJSON(anyMap1.ToJSON());
+  US_TEST_CONDITION((anyMap1_ToString == "{one : 1, two : 2}") || (anyMap1_ToString == "{two : 2, one : 1}"), "Any[std::unordered_map<std::string,int>].ToString()")
+  US_TEST_CONDITION((anyMap1_ToJSON == "{\"one\" : 1, \"two\" : 2}") || (anyMap1_ToJSON == "{\"two\" : 2, \"one\" : 1}"), "Any[std::unordered_map<std::string,int>].ToJSON()")
```

Comment:
Would it make sense to use the JSON reader (jsoncpp) on the output of ToJSON and verify the retrieved values from the parser are as expected.